server/config/config.go

```diff
@@ -292,7 +292,10 @@ func (c TLSConfig) Validate() error {
 
 // GetConfig returns a fully parsed Config object.
 func GetConfig(v *viper.Viper) Config {
-	cfg := config.GetConfig(v)
+	cfg, err := config.GetConfig(v)
```

Comment:
Please review the error check here.  There could be a better way to do this.

Reply:
Change this to return the error instead